css/properties/max-block-size.json

```diff
@@ -0,0 +1,70 @@
+{
+  "css": {
+    "properties": {
+      "-moz-binding": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/max-block-size",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
This is supported from Chrome 57/Opera 44. The status page says desktop only, but I think that is wrong. I'm confirming with the feature owner and will get back to you.

https://www.chromestatus.com/features/6325742262550528